RNAdam-core/src/main/scala/org/bdgenomics/RNAdam/algorithms/quantification/Tare.scala

```diff
@@ -132,4 +134,56 @@ object Tare extends Serializable {
 
     calibratedKmers
   }
+
+  /**
+   * Trains a linear regression model of the log of the transcript abundances
+   * versus the log of the transcript lengths.
+   * This is used to recalibrate the transcript abundances.
+   * Warning: This procedure assumes that the abundances are all > 0
+   * The log is undefined for numbers <= 0
+   *
+   * @param muHat An RDD containing the transcript abundances.
+   * @param tLen A mapping from transcript names to their lengths.
+   * @return The recalibrated transcript abundances.
+   */
+  def calibrateTxLenBias(muHat: RDD[(String, Double, Iterable[Long])],
+                         tLen: scala.collection.Map[String, Long]): RDD[(String, Double, Iterable[Long])] = {
+    // A local (not distributed) copy of muHat without the equivalence
+    // class ids. Removing these ids saves space since they are
+    // not used here.
+    val local: Array[(String, Double)] = muHat.map(mh => (mh._1, mh._2)).collect()
```

Comment:
Also, I will put sampling rate as a parameter, but should it have a default value? What should that be? Should it change depending on the data size?